lib/libutee/include/tee_syscall_numbers.h

```diff
@@ -61,21 +61,22 @@
 #define TEE_SCN_STORAGE_OBJ_TRUNC		52
 #define TEE_SCN_STORAGE_OBJ_SEEK		53
 #define TEE_SCN_CRYP_OBJ_GENERATE_KEY		54
-#define TEE_SCN_SE_SERVICE_OPEN			55
-#define TEE_SCN_SE_SERVICE_CLOSE		56
-#define TEE_SCN_SE_SERVICE_GET_READERS		57
-#define TEE_SCN_SE_READER_GET_PROP		58
-#define TEE_SCN_SE_READER_GET_NAME		59
-#define TEE_SCN_SE_READER_OPEN_SESSION		60
-#define TEE_SCN_SE_READER_CLOSE_SESSIONS	61
-#define TEE_SCN_SE_SESSION_IS_CLOSED		62
-#define TEE_SCN_SE_SESSION_GET_ATR		63
-#define TEE_SCN_SE_SESSION_OPEN_CHANNEL		64
-#define TEE_SCN_SE_SESSION_CLOSE		65
-#define TEE_SCN_SE_CHANNEL_SELECT_NEXT		66
-#define TEE_SCN_SE_CHANNEL_GET_SELECT_RESP	67
-#define TEE_SCN_SE_CHANNEL_TRANSMIT		68
-#define TEE_SCN_SE_CHANNEL_CLOSE		69
+/* Deprecated syscalls return TEE_ERROR_NOT_SUPPORTED */
+#define TEE_SCN_SE_SERVICE_OPEN__DEPRECATED		55
+#define TEE_SCN_SE_SERVICE_CLOSE__DEPRECATED		56
+#define TEE_SCN_SE_SERVICE_GET_READERS__DEPRECATED	57
+#define TEE_SCN_SE_READER_GET_PROP__DEPRECATED		58
+#define TEE_SCN_SE_READER_GET_NAME__DEPRECATED		59
+#define TEE_SCN_SE_READER_OPEN_SESSION__DEPRECATED	60
+#define TEE_SCN_SE_READER_CLOSE_SESSIONS__DEPRECATED	61
+#define TEE_SCN_SE_SESSION_IS_CLOSED__DEPRECATED	62
+#define TEE_SCN_SE_SESSION_GET_ATR__DEPRECATED		63
+#define TEE_SCN_SE_SESSION_OPEN_CHANNEL__DEPRECATED	64
+#define TEE_SCN_SE_SESSION_CLOSE__DEPRECATED		65
+#define TEE_SCN_SE_CHANNEL_SELECT_NEXT__DEPRECATED	66
+#define TEE_SCN_SE_CHANNEL_GET_SELECT_RESP__DEPRECATED	67
+#define TEE_SCN_SE_CHANNEL_TRANSMIT__DEPRECATED		68
+#define TEE_SCN_SE_CHANNEL_CLOSE__DEPRECATED		69
 #define TEE_SCN_CACHE_OPERATION			70
```

Comment:
We should have an empty line or a comment above this define.

Reply:
Will fix